src/main/scala/io/github/borisnaguet/sbt/plugins/CxfWsdl2JavaPlugin.scala

```diff
@@ -116,7 +116,7 @@ object CxfWsdl2JavaPlugin extends AutoPlugin {
         val cmd = Seq("java", "-cp", classpath) ++ sysProps ++ Seq("org.apache.cxf.tools.wsdlto.WSDLToJava") ++ args
 
         s.log.debug(cmd.toString())
-        cmd ! s.log
+        cmd.foreach(s.log(_))
```

Comment:
This should semantically be the same. sbt.Process is removed, and you should use scala.sys.Process instead, but importing that will make it not compiling, and this seemed like an easier solution.

Reply:
I didn't read the docs right on ProcessBuilder, this is not the same. So I'll have to fix this. I'll create a new PR with a fix for this during the next few days, sorry about this.